megatron/tokenizer/train_tokenizer.py

```diff
@@ -78,11 +78,12 @@ def train_tokenizer(
     tokenizer.train_from_iterator(json_iterator(input_dir), trainer)
 
     # And Save it
-    tokenizer.save(save_path, pretty=True)
-    print(f"Tokenizer saved at {save_path}")
+    if save_path:
```

Comment:
what's the rationale of adding this `if save_path` conditional?

Reply:
I was just trying to reduce the amount of IO in tests here.

More specifically, I set `--tokenizer_output_path ""` in `tests/unit/test_tokenizer.py` so that after training a new tokenizer we don't bother saving the output. The other option is to use a temporary output location so that the new model is saved & then cleaned up after the end of the unit test... if you would prefer the latter then that's totally fine by me.